changelog/6571.improvement.md

```diff
@@ -0,0 +1 @@
+Add a model relative path to CLI commands log 
```

Comment:
```suggestion
Log the model's relative path when using CLI commands.
```

Reply:
Done! Thank you for the suggestion @federicotdn